packages/dapp-connector-preact/package.json

```diff
@@ -0,0 +1,53 @@
+{
+	"name": "@cardano-sdk/dapp-connector-preact",
```

Comment:
it seems like there are 4 (or even 8?) spaces intentation here, can we change it to our default of 2?